app/src/main/java/com/jerboa/ui/components/common/SwipeToAction.kt

```diff
@@ -133,16 +140,23 @@ fun SwipeToAction(
 @Composable
 fun rememberSwipeActionState(
     swipeToActionPreset: SwipeToActionPreset,
-    onAction: (action: SwipeToActionType) -> Unit,
+    enableDownVotes: Boolean = true,
```

Comment:
Remove this default, its a footgun. We have had many cases before, we forgot to fill it in.

Reply:
You're right, fixed it